websocket/polygon.go

```diff
@@ -448,11 +464,10 @@ func getParams(market Market, topic Topic, tickers ...string) (string, error) {
 }
 
 func (c *Client) setSubscriptions(topic Topic, tickers ...string) {
+	if tickers[0] == "*" {
```

Comment:
this could panic if `len(tickers) == 0` btw. not that much of an issue since it's internal logic but might want to guard against it anyway (i.e. `if len(tickers) > 0 && tickers[0] == "*"`)